deps/rabbit/src/rabbit_fifo_client.erl

```diff
@@ -738,7 +738,9 @@ handle_delivery(Leader, {delivery, Tag, [{FstId, _} | _] = IdMsgs},
                        consumer_deliveries = CDels0} = State0) ->
     QRef = qref(Leader),
     {LastId, _} = lists:last(IdMsgs),
-    Consumer = #consumer{ack = Ack} = maps:get(Tag, CDels0),
+    %% Note:
+    %% https://github.com/rabbitmq/rabbitmq-server/issues/3729
```

Comment:
Please capture the general idea behind the ticket. This URL might not work anymore in 10 years.